### PR TITLE
LibWeb: Add support for text in the `AffineCommandExecutorCPU`

### PR DESCRIPTION
**Pull request by MacDue**
**Demo:**
image

https://github.com/SerenityOS/serenity/assets/11597044/83662807-688b-4ce7-bc89-5ec0910ebd7c

### CODE REVIEW

**Comment by MacDue** [comment]
These are my last few commits a week or so ago. Probably won't continue with this experiment anymore (as transforms are likely not needed for the Serenity-only LibWeb)... but it was fun while it lasted :)

File: Userland/Libraries/LibGfx/Path.cpp [hunk 1]
```diff
@@ -417,6 +417,12 @@ Path Path::copy_transformed(Gfx::AffineTransform const& transform) const
     return result;
 }
 
+void Path::transform(AffineTransform const& transform)
+{
+    for (auto& point : m_points)
```
Comment (nico) [inline Userland/Libraries/LibGfx/Path.cpp:422]: Isn't changing in-place and caching paths a somewhat dangerous combination?

**Review by nico — COMMENTED** [review]
I think merging this would be great! And who knows, maybe you'll continue it one day :)

A few test failures, otherwise looks great:

https://github.com/SerenityOS/serenity/actions/runs/9439428736/job/25997539818?pr=24556

```
Fail: /Users/runner/work/serenity/serenity/Tests/LibWeb/Ref/svg-text-effects.html
Fail: /Users/runner/work/serenity/serenity/Tests/LibWeb/Ref/canvas-text.html
```

Might need rebaselining or something? On https://github.com/SerenityOS/serenity/actions/runs/9439428736/job/25997539866?pr=24556 too (same tests).

File: Userland/Libraries/LibGfx/Path.cpp (hunk 1 above)
Comment (MacDue) [reply Userland/Libraries/LibGfx/Path.cpp:422]: The cache returns a `Path const&`, so to use `.transform()` on the path you'd have to do an evil `const_cast`.

**Comment by nico** [comment]
fddffdfdwfsf!

**Pull request merged**